lib/src/runner/configuration/load.dart

```diff
@@ -83,7 +81,41 @@ class _ConfigurationLoader {
   Configuration load() => _loadGlobalTestConfig()
       .merge(_loadLocalTestConfig())
       .merge(_loadGlobalRunnerConfig())
-      .merge(_loadLocalRunnerConfig());
+      .merge(_loadLocalRunnerConfig())
+      .merge(_loadIncludeConfig());
```

Comment:
Merging this last means the included config will override the local config, which isn't what we want.

This also makes me realize we should test that the local config takes precedence when they can't be merged.

Reply:
Can you help me understand this?

I can move the `_loadIncludeConfig` after `_loadGlobalTestConfig`.

Then what? What should I test?

Reply:
I think `_loadIncludeConfig()` should come first in the list, so that all other configuration overrides it.

You could test that

```yaml
# dart_test.yaml
include: other.yaml
concurrency: 5
```

```yaml
# other.yaml
concurrency: 10
```

ends up with concurrency 5.

Reply:
Thanks! Done!